keygen/src/keygen.rs

```diff
@@ -349,7 +388,47 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                         .validator(is_parsable::<usize>)
                         .default_value(&default_num_threads)
                         .help("Specify the number of grind threads"),
-                ),
+                )
+                .arg(
+                    Arg::with_name("mnemonic")
+                        .long("mnemonic")
+                        .alias("use-mnemonic")
+                        .help("Generate using a mnemonic key phrase"),
+                )
+                .arg(
+                    Arg::with_name("word_count")
+                        .requires_all(&["mnemonic"])
+                        .long("word-count")
+                        .possible_values(&["12", "15", "18", "21", "24"])
+                        .default_value("12")
+                        .value_name("NUMBER")
+                        .takes_value(true)
+                        .help("(requires mnemonic) Specify the number of words that will be present in the generated seed phrase"),
+                )
+                .arg(
+                    Arg::with_name("language")
+                        .requires_all(&["mnemonic"])
+                        .long("language")
+                        .possible_values(&["english", "chinese-simplified", "chinese-traditional", "japanese", "spanish", "korean", "french", "italian"])
+                        .default_value("english")
+                        .value_name("LANGUAGE")
+                        .takes_value(true)
+                        .help("(requires mnemonic) Specify the mnemonic lanaguage that will be present in the generated seed phrase"),
+                )
+                .arg(
+                    Arg::with_name("no_passphrase")
+                        .requires_all(&["mnemonic"])
+                        .long("no-bip39-passphrase")
+                        .alias("no-passphrase")
+                        .help("(requires mnemonic) Do not prompt for a BIP39 passphrase"),
+                )
+                .arg(
+                    Arg::with_name("no_outfile")
+                        .requires_all(&["mnemonic"])
+                        .long("no-outfile")
+                        .conflicts_with_all(&["outfile", "silent"])
```

Comment:
But grind produces an outfile, but it can also now produce a mnemonic.  If you are producing a mnemonic, you don't technically need an outfile since you can re-generate it from solana-keygen recover.  For example if I personally am going to use this command to grind out vanity account names for use as system accounts, I typically would want them as a paper wallet which means I never even want them to exist as a file.  So I would use the --no-outfile argument here.

Reply:
Sorry, I mean that just this line `.conflicts_with_all(&["outfile", "silent"])` is unneeded, since grind doesn't have those parameters

Reply:
Oops, copy-paste error.  Embarrassing ...